ui/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -231,6 +256,52 @@ export default class SelectHardware extends Component {
     );
   }
 
+  renderQRHardwareWalletSteps() {
+    const steps = [];
+    steps.push({
+      title: this.context.t('QRHardwareWalletSteps1Title'),
+      message: this.context.t('QRHardwareWalletSteps1Description'),
+    });
+
+    steps.push({
+      message: (
+        <p>
```

Comment:
Sorry for not pointing this out earlier; instead of using `<p>` tags, we use our Typography component:

```
<Typography variant={TYPOGRAPHY.Paragraph}>
```